SimpleEspNowConnection missing return value

It is set to return a bool but never returns anything, which breaks compilation
```
bool SimpleEspNowConnection::DeviceMessageBuffer::deleteBuffer(const uint8_t *device, long id)
{
    for(int i = 0;i<MaxBufferSize; i++)
    {
      if(_dbo[i] != NULL && memcmp(_dbo[i]->_device, device, 6) == 0 && _dbo[i]->_id == id)
      {
        delete _dbo[i];
        _dbo[i] = NULL;
      }
    }	
}
```

```
.../Arduino/libraries/SimpleEspNowConnection/src/SimpleEspNowConnection.cpp: In member function 'bool SimpleEspNowConnection::DeviceMessageBuffer::deleteBuffer(const uint8_t*, long int)':
.../Arduino/libraries/SimpleEspNowConnection/src/SimpleEspNowConnection.cpp:195:1: error: no return statement in function returning non-void [-Werror=return-type]
  195 | }
      | ^
cc1plus: some warnings being treated as errors

exit status 1

Compilation error: exit status 1
```

Thanks

--- a/src/SimpleEspNowConnection.cpp
+++ b/src/SimpleEspNowConnection.cpp
@@ -191,7 +191,8 @@ bool SimpleEspNowConnection::DeviceMessageBuffer::deleteBuffer(const uint8_t *de
         delete _dbo[i];
         _dbo[i] = NULL;
       }
-    }	
+    }
+    return true;
 }
 
 bool SimpleEspNowConnection::DeviceMessageBuffer::deleteBuffer(SimpleEspNowConnection::DeviceMessageBuffer::DeviceBufferObject* dbo)

